src/sx-tailwind/src/__tests__/sxGenerator.test.js

```diff
@@ -163,44 +168,45 @@ it('does not remove rem units from font-size', () => {
   `);
 });
 
-it('does not overwrite declarations', () => {
+it('does overwrite declarations', async () => {
   const css = `.bg-black {
     --bg-opacity: 1;
     background-color: #000;
     background-color: rgba(0, 0, 0, var(--bg-opacity));
   }`;
 
-  expect(generate(css)).toMatchInlineSnapshot(`
+  expect(await convertToSx(css, config)).toMatchInlineSnapshot(`
     Object {
       "bg-black": Object {
-        "backgroundColor": "#000",
+        "--bg-opacity": "1",
+        "backgroundColor": "rgba(0, 0, 0, var(--bg-opacity))",
       },
     }
   `);
 });
 
-it('skips empty styles', () => {
+it('skips empty styles', async () => {
   const css = `.hover\\:empty-style:hover {}`;
 
-  expect(generate(css)).toMatchInlineSnapshot(`Object {}`);
+  expect(await convertToSx(css, config)).toMatchInlineSnapshot(`Object {}`);
 });
 
-it('skips empty media queries', () => {
+it('skips empty media queries', async () => {
```

Comment:
This is testing keyframes, but the it text says media queries

Reply:
Renamed to "skips unsupported at-rules"